### PR TITLE
use setAllValue in the form instead of patchAllValue

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

File: libs/movie/src/lib/movie/form/movie.form.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ import { createCredit, Stakeholder, createStakeholder, Director } from '@blockfr
 import { createMovieAppAccess } from '@blockframes/utils/apps';
 import { toDate } from '@blockframes/utils/helpers';
 import { Languages } from '@blockframes/utils/static-model';
+import { keys } from 'lodash';
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/form/movie.form.ts:62]: back in the good old days, I heard a french guy say: `you shall not use lodash in frontend code`... he almost got the same beard as you...

Comment (fritzschoff) [reply libs/movie/src/lib/movie/form/movie.form.ts:62]: btw, am I blind or do you don't use it?

File: libs/movie/src/lib/movie/form/shell/shell.component.ts [hunk 2]
```diff
@@ -167,11 +165,11 @@ export class MovieFormShellComponent implements TunnelRoot, OnInit, AfterViewIni
         inline: 'start',
       });
     });
-    this.sub.push(routerSub);
+    this.sub.add(routerSub);
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/form/shell/shell.component.ts:168]: whoop whoop

**Review by fritzschoff — APPROVED** [review]

File: libs/movie/src/lib/movie/form/movie.form.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply libs/movie/src/lib/movie/form/movie.form.ts:62]: WTF it is doing here...

**Pull request merged**